app/Http/Controllers/Auth/VerificationController.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+namespace App\Http\Controllers\Auth;
+
+use App\Http\Controllers\Controller;
+use App\Providers\RouteServiceProvider;
+use Illuminate\Foundation\Auth\VerifiesEmails;
+
+class VerificationController extends Controller
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Missing doc comment for class VerificationController](https://app.codacy.com/gh/BuildForSDG/Team-059-Product/pullRequest?prid=5478046)